### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by sebasptsch**

### CODE REVIEW

File: packages/backend/src/index.ts [hunk 1]
```diff
@@ -119,6 +119,8 @@ await server.setNotFoundHandler((req, res) => {
 // Set the server to listen on port 3000
 await server.listen({ port: 3000, host: "0.0.0.0" });
 
+process.send?.("ready");
```
Comment (ShaanCoding) [inline packages/backend/src/index.ts:122]: What does this do?

File: packages/backend/test/e2e.test.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+afterAll(async () => {
+    childProcessInstance.kill();
+});
+
+describe("auth", () => {
+  const testUser = {
+    password: faker.internet.password(),
+    email: faker.internet.email(),
+    name: faker.internet.userName(),
+    phone: "1234567890",
```
Comment (ShaanCoding) [inline packages/backend/test/e2e.test.ts:55]: Awesome stuff

**Review by ShaanCoding — APPROVED** [review]

File: packages/backend/src/index.ts (hunk 1 above)
Comment (sebasptsch) [reply packages/backend/src/index.ts:122]: Marks the process as ready for testing so we know the server is ready to receive messages

**Comment by ShaanCoding** [comment]
Overall LGTM, should be good to merge. Just had one question but it's more of curiosity if anything, feel free to merge when convenient.

image

File: packages/backend/src/index.ts (hunk 1 above)
Comment (ShaanCoding) [reply packages/backend/src/index.ts:122]: Setup for vitest config, makes sense. Thanks

**Pull request merged**